GCTC-NTGC/gc-digital-talent
# Warning: Can't update component UserProfilePrintButton

Resolves #4878.

The View User page had 3 child components making their own api queries for largely the same data. This makes the ViewUser component do one query for all relevant data and pass that data to the subcomponents.

To test: 
1. go to `http://localhost:8000/en/admin/users`
2. pick a user and click "View"
3. open your browser console
4. click the "Candidate Profile" tab
5. confirm the "Cannot update a component" warning doesn't show up.
6. confirm both tabs and pdf generated by the "Print Profile" button all appear to be receiving all necessary data.
